app/controllers/api/v2/app_identifier_visitor_configs_controller.rb

```diff
@@ -0,0 +1,26 @@
+class Api::V2::AppIdentifierVisitorConfigsController < UnauthenticatedApiController
+  include CorsSupport
+
+  def show # rubocop:disable Metrics/AbcSize
+    build_path = AppVersionBuildPath.new(build_params)
+    if build_path.valid?
+      app_build = AppVersionBuildPath.new(build_params).app_build
```

Comment:
why not `build_path.app_build`?

Reply:
great question! i dunno. this was a copy-paste job, but i went in and fixed it in all 4 places.